packages/camera/lib/camera.dart

```diff
@@ -281,11 +308,11 @@ class CameraController extends ValueNotifier<CameraValue> {
       return new Future<Null>.value(null);
     } else {
       return _creatingCompleter.future.then((_) async {
-        await _eventSubscription?.cancel();
         await _channel.invokeMethod(
-          'dispose',
+          'closeCamera',
```

Comment:
We use `dispose` in several other plugins for cleaning up resources. I think we want to stick with that.